docs/programming-oak.md

```diff
@@ -262,6 +262,7 @@ name = "hello_world"
 [modules]
 app = { path = "examples/hello_world/bin/hello_world.wasm" }
 translator = { path = "examples/hello_world/bin/translator.wasm" }
+
```

Comment:
These blank lines seem odd, are they added by prettier? Why only here?

Reply:
It's the TOML plugin we use, it always adds an empty line at the end of toml files. Previously it would skip it for toml excerpts in markdown files, but it seems that's no longer the case.

Reply:
Which TOML plugin do we use? :eyes:

Reply:
It's a bit strange, but unfortunately it seems that plugin is no longer maintained. Imho admittedly strange formatting, but probably not worth staying on the previous prettier version to avoid this

Reply:
do you mean that prettier is also formatting code inside code blocks if it has a plugin for that particular file format?

Reply:
but also, the version of the prettier-toml plugin has not changed, so I'm even more confused

Reply:
[As of 2.1 prettier does apply formatting to code blocks it recognizes](https://prettier.io/blog/2020/08/24/2.1.0.html#add---embedded-language-formattingautooff-option-7875httpsgithubcomprettierprettierpull7875-by-bakkothttpsgithubcombakkot-8825httpsgithubcomprettierprettierpull8825-by-fiskerhttpsgithubcomfiskers). Hence previously the plugin was not applied to the code blocks, but since prettier was updated it now is.

Reply:
We can turn off this option (formatting code-blocks) if desired.

Reply:
Ah cool, thanks for the link, that's interesting! Happy to keep as it is